drivers/tty/serdev/pibridge_trace.h

```diff
@@ -13,7 +13,16 @@
 #define PIBRIDGE_TRACE_MAX_FIFO_DUMP		128
 #define PIBRIDGE_TRACE_MAX_PACKET_DATA_LEN	256
 
-/*  Header */
+/*
+ * pibridge_gate_header_class
```

Comment:
It would be good to have a brief description here for each class, e.g. "print header of gateway protocol telegrams" (see kernel-doc-formatted example below).

Reply:
Ok

Reply:
The description for the events itself is IMHO already in the "info" part of the comment. However for the event classes I added a descriptive sentence each.